README.md

```diff
@@ -75,6 +75,15 @@ gradle run --args="integ-test -t=ECS_TEST --package-version={the version you wan
 gradle run --args="clean -t=ECSClean --package-version={the version you want to test}"
 ```
 
+### Run EKS Integ-test on Sidecar mode (EMF Metrics)
+```
+gradle run --args="integ-test -t=EKS_TEST --package-version={the version you want to test} -k eksClusterName=my-cluster-name -k awsAuthenticatorPath=/my/authenticator/path"
```

Comment:
can we say ```awsAuthenticatorPath=/my/authenticator/path``` is optional? I see you will download the default one from aws  if customer doesn't provide it.

Reply:
yes, so optional option can be absent here, correct?

Reply:
yes, can we add a little description for running this cmd? thx